src/components/slider/slider.component.html

```diff
@@ -4,30 +4,49 @@
     <div class="track-section track-lower" [style.flex-grow]="tracks.lower.size" [style.background]="tracks.lower.color"></div>
 
     <!-- Lower Thumb Button / Line -->
-    <div class="thumb lower" uxDrag
-        [style.left.%]="thumbs.lower.position" 
-        [class.active]="thumbs.lower.drag" 
-        [style.z-index]="thumbs.lower.order" 
+    <div class="thumb lower"
+        uxDrag
+        role="slider"
+        tabindex="0"
+        #lowerthumb
+        [attr.aria-label]="options.type === sliderType.Range ? 'Slider minimum value' : 'Slider value'"
```

Comment:
"Minimum value" in this label might be a bit misleading - maybe "lower value" and "upper value"?